mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -112,34 +112,77 @@ export function DexScreen (): JSX.Element {
     })
   }
 
+  // sorting functions
+  const availablePairsSortingFn = (firstPair: DexItem<PoolPairData>, secondPair: DexItem<PoolPairData>): number => (
+    new BigNumber(secondPair.data.totalLiquidity.usd ?? 0).minus(firstPair.data.totalLiquidity.usd ?? 0).toNumber() ??
+    new BigNumber(secondPair.data.id).minus(firstPair.data.id).toNumber()
+  )
+  const yourPairsSortingFn = (firstPair: DexItem<WalletToken>, secondPair: DexItem<WalletToken>): number => (
+    pairs.findIndex(x => x.data.id === firstPair.data.id) -
```

Comment:
I have just clarified with @pierregee that was previously working on this logic, the sorting for `Your pool pairs` was meant to follow the order of `id` in the pairs sorted in `Available pool pairs`.

```suggestion
    filteredAvailablePairs.findIndex(x => x.data.id === firstPair.data.id) -
```

Reply:
Thanks for the clarification.
I implemented a fix.
I introduced an intermediate variable called sortedAvailablePairs which is never filtered to base the sorting on.